address_test.go

```diff
@@ -64,6 +84,24 @@ func TestAddresses(t *testing.T) {
 			},
 			net: &chaincfg.MainNetParams,
 		},
+		{
+			name:    "custom mainnet p2pkh",
```

Comment:
update names to include ltc or litecoin

Reply:
@cfromknecht is that ok to include ltc/litecoin naming in this lib?

Reply:
i'd say it's fine, for the purposes of demonstrating that this method works for verifying addrs that use constants not defined by the library